episodes/03-create.md

```diff
@@ -116,33 +101,33 @@ wording of the output might be slightly different.
 
 ## Places to Create Git Repositories
 
-Along with tracking information about planets (the project we have already created),
-Dracula would also like to track information about moons.
-Despite Wolfman's concerns, Dracula creates a `moons` project inside his `planets`
+Along with tracking information about recipes (the project we have already created),
+Alfredo would also like to track information about desserts specifically.
+Despite Jimmy's concerns, Alfredo creates a `desserts` project inside his `recipes`
```

Comment:
```suggestion
Alfredo creates a `desserts` project inside his `recipes`
```

This was a joke about Wolfman's reaction to moonlight that no longer makes sense in the context of the new narrative.